packages/cli/src/controller/deploy-controller.ts

```diff
@@ -57,7 +58,7 @@ export async function createDeployment(
     );
     return res.data.deployment;
   } catch (e) {
-    errorHandle(e, 'Error deploying to hosted service:');
+    throw errorHandle(e, 'Error deploying to hosted service:');
```

Comment:
`errorHandle` says the return type is error but it just throws. Do you think we could update it so it returns errors rather than throwing?